### PR TITLE
FACT-1676 updating frontend readme

### PR DESCRIPTION
**Pull request by justiceia**
### JIRA link ###
https://tools.hmcts.net/jira/browse/FACT-1676

### Change description ###
Can't find much that needs adding but added contents

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,25 @@
 
 Website to find a court or tribunal.
 
-## Getting Started..
-
-### Prerequisites.
+## Table of Contents
+
+* [Getting Started](#setting-started)
```
Comment (amstevenson) [inline README.md:7]: Probably being slow, are these linking to something further down in the readme?

File: README.md [hunk 2]
```diff
@@ -2,9 +2,25 @@
...
+  * [Code style](#code-style)
+  * [Updating SCSS and JavaScript](#updating-scss-and-javascript)
+  * [Running the tests](#running-the-tests)
+  * [Security](#security)
+    * [CSRF prevention](#csrf-prevention)
+    * [Helmet](#helmet)
+  * [Healthcheck](#healthcheck)
+* [License](#license)
+
+## Getting Started
```
Comment (amstevenson) [inline README.md:21]: This section looks empty

File: README.md (hunk 1 above)
Comment (justiceia) [reply README.md:7]: I think that was just the title section for the 3 subsections below it. The bits to get started are in the subsections i think.

File: README.md (hunk 2 above)
Comment (justiceia) [reply README.md:21]: I think that was just the title section for the 3 subsections below it. The bits to get started are in the subsections i think.

File: README.md (hunk 1 above)
Comment (justiceia) [reply README.md:7]: ah i had a spelling mistake

**Review by amstevenson — APPROVED** [review]

**Pull request merged**